src/store/middlewares/login.js

```diff
@@ -17,30 +14,29 @@ const loginMiddleware = store => next => (action) => { // eslint-disable-line ma
           data: { isHarwareWalletConnected: false },
```

Comment:
you don't need to store isHarwareWalletConnected value in the redux store because it doesn't use in any presetional component. So, dispatching settingsUpdated action here is completely unnecessary.

Reply:
Of course, it is used in a presentational component. Specifically here:
https://github.com/LiskHQ/lisk-hub/blob/1736e98a13b8b5cc78207f8c2db5ebf5fd346f31/src/components/loginV2/loginV2.js#L276

Reply:
still, it doesn't make sense to store this value in the setting reducer.

Reply:
I agree, but it's way beyond the scope of this PR.